cron/.env.example

```diff
@@ -0,0 +1,2 @@
+DATA_INPUT_DB_URL="postgresql://doadmin:l5al4hwte8qmj6x8@db-postgresql-sfo2-nextgen-do-user-1067699-0.db.ondigitalocean.com:25060/data_pipeline_dev?ssl=true"
```

Comment:
Looks like the actual dev database db url made it in here.  Not a huge deal since we don't carefully control this one, but not a practice we want volunteers to this is ok.

Reply:
I apologize for this. Since I saw sqs url, I mistook it is ok to leave dev db urls here. Will remove it.